README.md

```diff
@@ -48,36 +38,32 @@ class AppComponent {
 
 See the overview for a [quick start](https://vaadin.com/docs/-/part/elements/angular2-polymer/overview.html).
```

Comment:
Docs on vaadin.com are outdated and could be unpublished soon. Let’s change the documentation links to point to this repo instead.